FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Alok Subedi on 13/01/2021.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+class FeedImageMapper{
+	private struct root: Decodable{
```

Comment:
As a convention, type names start with a capital letter:

```suggestion
	private struct Root: Decodable {
```